.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,60 @@
+name: Weekly maintenance/update tasks
+on:
+    # workflow_dispatch allows for manual triggering of the workflow
+    # Useful for testing that it works without waiting on the cron schedule.
+    workflow_dispatch:
+    schedule:
+        # https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
+        #       ┌─────────── minute (0 - 59)
+        #       │  ┌───────── hour (0 - 23)
+        #       │  │ ┌─────── day of the month (1 - 31)
+        #       │  │ │ ┌───── month (1 - 12 or JAN-DEC)
+        #       │  │ │ │ ┌─── day of the week (0 - 6 or SUN-SAT)
+        #       v  v v v v
+        - cron: 30 1 * * 1
```

Comment:
Thank you for documenting this! :)

How did you choose 7:30 AM? (i.e. why not 7 AM?)

Reply:
Props to @somewhatabstract for the prompt and source of the docs. I really like the format!

I probably overthought this by a bunch.  I picked a time that would be at night in North America (considering East Coast and West Coast time zones).  Then I picked a time not at the top of the hour because tasks are often scheduled at the top of the hour and this would avoid adding to a potential thundering herd. In the end, it's probably much overthought! :)

Reply:
And oh no!  I think I've got it at 1:30 UTC right now (contradicting the doc string)!  I'll fix.